sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableChange.java

```diff
@@ -726,22 +746,29 @@ public String[] fieldNames() {
      * is not provided as updating column default values does not need to back-fill existing data.
      * Empty string means dropping the column default value.
      */
-    public String newDefaultValue() { return newDefaultValue; }
+    public String newDefaultValue() {
```

Comment:
shall we mark this function as deprecated and encourage users to call `newCurrentDefault`?